server/database/dataBradley.sql

```diff
@@ -177,8 +178,12 @@ insert into users(first_name,last_name,email,password,other_skills,job_descripti
 
   insert into questions(question,date,owner,speclalization_id)
   values
-  ('Have you ever been in a hospital  ?','2018-10-8','1','1'),
-  ('Where is the pain exactly  ?','2018-3-8','1','5');
+  ('Where is the pain exactly  ?','2018-3-8','1','1'),
+  ('Where is the pain exactly  ?','2018-3-8','1','2'),
+  ('Where is the pain exactly  ?','2018-3-8','1','3'),
+  ('Where is the pain exactly  ?','2018-3-8','1','4'),
+  ('Where is the pain exactly  ?','2018-3-8','1','5'),
+  ('Where is the pain exactly  ?','2018-3-8','1','6');
```

Comment:
@ali-7 
This might not be the most important thing to worry about right now, but this would read better if there were no spaces between "exactly" and the question mark.  So change to `'Where is the pain exactly?'`.  Maybe it makes sense to do these copy fixes all at once at a later date though.

Reply:
@tomduggan85 actually you mentioned it to me before, and i remember that i modified theses lines :smiley: , maybe i copied the wrong line, i'll change it later.
thanks